### PR TITLE
htop: add Japanese translation

### PR DESCRIPTION
**Pull request by yuiseki**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** Unknown

### CODE REVIEW

File: pages.ja/common/htop.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# htop
+
+> 実行中のプロセスに関する動的なリアルタイム情報を表示する。 `top` の拡張版。
+> 詳しくは: <https://htop.dev/>.
```
Comment (kbdharun) [inline pages.ja/common/htop.md:4]:
```suggestion
> 詳しくはこちら: <https://htop.dev/>.
```

https://github.com/tldr-pages/tldr/blob/0277a1eec19204479a3572b74047467de69c58dd/scripts/set-more-info-link.py#L25

This is the standard work we use across all our pages.

File: pages.ja/common/htop.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+- htop を起動:
+
+`htop`
+
+- 特定のユーザが所有するプロセスを表示する htop を起動する:
+
+`htop --user {{username}}`
```
Comment (kbdharun) [inline pages.ja/common/htop.md:12]:
```suggestion
`htop --user {{ユーザー名}}`
```
[Ref](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md)

File: pages.ja/common/htop.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+- 指定した `sort_item` でプロセスをソートする (利用可能なオプションは `htop --sort help` を使用する):
+
+`htop --sort {{sort_item}}`
```
Comment (kbdharun) [inline pages.ja/common/htop.md:16]:
```suggestion
`htop --sort {{並べ替え項目}}`
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

**Tip:** Whenever possible try to translate the placeholders too as it will make them better readable to native speakers.

File: pages.ja/common/htop.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+- htop を起動:
+
+`htop`
+
+- 特定のユーザが所有するプロセスを表示する htop を起動する:
+
+`htop --user {{ユーザー名}}`
+
+- 指定した `sort_item` でプロセスをソートする (利用可能なオプションは `htop --sort help` を使用する):
```
Comment (yuiseki) [inline pages.ja/common/htop.md:14]:
```suggestion
- 指定した `並べ替え項目` でプロセスをソートする (利用可能なオプションは `htop --sort help` を使用する):
```

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Review by blueskyson — APPROVED** [review]

**Pull request merged**